src/convoRouter.js

```diff
@@ -12,14 +12,33 @@ router.post('/me/messages', (req, res) => {
     req_query: req.query,
   })}`);
 
-  if (_.isEqual(req.query.access_token, ACCESS_TOKEN)) {
-    res.json({
+  const authz = req.get('Authorization');
+  console.log('Authorization header value = ', authz);
```

Comment:
I know it's a reference implementation, but should we be logging this if it's valid?